### PR TITLE
Apply the new design for Two-page

### PR DESCRIPTION
**Pull request by joyl1216**
Screenshot_1605573378
Screenshot_1605576297
Screenshot_1605576303

### CODE REVIEW

**Review by CesarValiente — CHANGES_REQUESTED** [review]
@joyl1216 Is this branch meant to be merged into `master` directly instead of in `develop`?

Also I see that there are changes regarding `CompanionPane` too here instead just about `TwoPage` pattern. Is this desired or a mistake?

File: TwoPage/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,4 +6,6 @@
 
 <resources>
     <string name="app_name">Two Page</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (bimiron) [inline TwoPage/src/main/res/values/strings.xml:10]: Should we do something about this TODO? 😃

**Comment by joyl1216** [comment]
@CesarValiente Yes, you are right. It is my bad. The PR should go into `Develop`, not `Master`. Is there a way to set `Develop` as the default branch, like what we did before in ADO, to avoid the same mistake?

**Comment by Mehul1995** [comment]
@joyl1216, One small change. In the two page view I want the page number on the right screen to be right-aligned rather a left-aligned; So that it mimics just like a book mode. 

Like the image here: 
image

**Comment by joyl1216** [comment]
Updated. 
Screenshot_1605661308

**Review by Mehul1995 — APPROVED** [review]
Looks good to me!

File: TwoPage/src/main/res/layout/fragment_first_page.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        android:id="@+id/textView3"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:fontFamily="serif"
+        android:textColor="@color/black"
+        android:lineSpacingExtra="5dp"
+        android:text="Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source."
```
Comment (CesarValiente) [inline TwoPage/src/main/res/layout/fragment_first_page.xml:62]: We probably want to move strings to the `strings.xml` file so they are easier to be handled and updated.
(this applies to all view strings -> android:text)

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Joy for addressing the changes!

**Pull request merged**